.gitignore

```diff
@@ -8,4 +8,5 @@
 /static/specifications
 /layouts/robots.txt
 /.project
-/data/eclipsefdn_projects.yaml
\ No newline at end of file
+/data/eclipsefdn_projects.yaml
+package-lock.json
```

Comment:
Please make sure to include an updated package-lock file after removing this line.

This file will update by simply running npm install.

Reply:
Someone else already removed it from repo

Reply:
More info: https://docs.npmjs.com/cli/v6/configuring-npm/package-lock-json

Reply:
https://github.com/jakartaee/jakarta.ee

Reply:
You are right, we deleted that file yesterday when we merge the jakarta9 landing page. 

Let's bring this back since this allows developers to install the exact version that was used to deploy the site. I can also add it after merging this PR.

Reply:
no no, I'm ok with adding it back 👍

Reply:
done